internal/hcl/block.go

```diff
@@ -49,6 +49,9 @@ var (
 				Type:       "data",
 				LabelNames: []string{"type", "name"},
 			},
+			{
```

Comment:
I think maybe a better solution would be to change `loadBlocksFromFile` to use `PartialContent` over `Content`, as we don't actually support type `moved` https://github.com/infracost/infracost/blob/2d4a16fe989b9c363f16f4295dd4466e85c4fde4/internal/hcl/block.go#L987

e.g:

```go
	contents, _, diags := file.hclFile.Body.PartialContent(schema)
	if diags != nil && diags.HasErrors() {
		return nil, diags
	}
```

PartialContent is like Content except that it permits the configuration to contain additional blocks or attributes not specified in the schema. This means we don't break in the future on new block type additions.